ui/app/rif/components/modal/addNewTokenNetworkAddress/addNewTokenNetworkAddress.js

```diff
@@ -56,16 +56,16 @@ class AddNewTokenNetworkAddress extends Component {
 					}}
 				>
 					<FontAwesomeIcon className={'add-new-multicrypto-select-value-icon'} icon={option.icon} color={option.color}/>
-					<span>&nbsp;{option.label}</span>
+					<span className={'label-spacing-left'}>{option.label}</span>
```

Comment:
You can use direct naming here, className="label-spacing-left", there is not need to use react bindings. It's a matter of taste anyway.